portfolio/src/main/webapp/index.html

```diff
@@ -168,28 +168,36 @@ <h4 class="experience_header">Past Positions</h4>
 
                 <p>Name: </p>
                 <input type="text" name="comment_name">
-                <!-- <p>Comment: <input type="text" class="long_text" name="comment_content"></p> -->
                 <p>Comment: </p>
                 <textarea type="text" name="comment_content" rows="2"></textarea>
 
                 <input type="submit" />
             </form>
             <hr>
-            <!-- <div id="comments_container"> -->
-                <!-- <ul id="comment-list">
-                    <li class="comment"><span><strong>Andrew Hojel</strong></span><button class="comment_button delete"><i class="fa fa-trash"></i></button><br/><span>Let's try this Let's try this Let's try this Let's try this Let's try this Let's try thisLet's try this Let's try thisLet's try this vLet's try this</span></li> 
-                </ul> -->
-            <!-- </div> -->
+            <div>
+                <label for="comment_count">Number of Comments Displayed: </label>
+                <select name="comment_count" id="comment_count" onchange="getComments()">
+                    <option value=5>5</option>
+                    <option value=10 selected>10</option>
+                    <option value=25>25</option>
+                    <option value=50>50</option>
+                </select>
+                <button class="comment_button delete_all" onclick="deleteAllComments()">Delete All</button>
+            </div>
+            <div class="sort_div">
+                <label style="margin-top: 50px" for="comment_sorting">Sorting Order: </label>
```

Comment:
inline styles are hard to maintain and generally considered bad in most cases. can you add an id to the label and move the style to css?